### PR TITLE
ui: clarify disable_irreversible action

### PR DESCRIPTION
**Pull request by LasTshaMAN**
Resolves https://github.com/decred/dcrdex/issues/1631

### CODE REVIEW

File: client/webserver/locales/pl-pl.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ var PlPL = map[string]string{
 	"export_app_pw_msg":              "Podaj hasło aplikacji, aby potwierdzić eksport konta dla",
 	"Disable Account":                "Zablokuj konto",
 	"disable_app_pw_msg":             "Podaj hasło aplikacji, aby zablokowac konto",
-	"disable_irreversible":           `<span class="red">Uwaga:</span> Ta czynność jest nieodwracalna - zablokowanego konta nie można później odblokować.`,
```
Comment (chappjc) [inline client/webserver/locales/pl-pl.go:43]: Are you using google translate for these?  I will ping the people who did the original translations assuming you don't speak pl, pt, and zh.  :)

Comment (LasTshaMAN) [reply client/webserver/locales/pl-pl.go:43]: yeah, I should have mentioned that its from Google translate :)

just didn't think you have dedicated people to keep it updated, by all means, let them know and I'll incorporate their feedback here

Comment (chappjc) [reply client/webserver/locales/pl-pl.go:43]: @artikozel how does this PL translation look?

File: client/webserver/locales/zh-cn.go [hunk 2]
```diff
@@ -40,7 +40,7 @@ var ZhCN = map[string]string{
 	"export_app_pw_msg":              "输入密码以确认帐户导出",
 	"Disable Account":                "禁用帐户",
 	"disable_app_pw_msg":             "输入您的密码以禁用帐户",
-	"disable_irreversible":           `<span class="red">注意：</span>此操作是不可逆的 - 帐户停用后将无法再重新激活。`,
+	"disable_dex_server":             "该 DEX 服务器可能会在未来任何时候通过再次添加重新启用（您无需支付费用）。",
```
Comment (chappjc) [inline client/webserver/locales/zh-cn.go:43]: @DominicTing how does this ZH translation look?

"This DEX server may be re-enabled (you won't have to pay the fee) at any time in the future by adding it again."

File: client/webserver/locales/pt-br.go [hunk 3]
```diff
@@ -40,7 +40,7 @@ var PtBr = map[string]string{
 	"export_app_pw_msg":              "Informe a senha para confirmar exportação de conta",
 	"Disable Account":                "Desativar Conta",
 	"disable_app_pw_msg":             "Informe sua senha para desativar conta",
-	"disable_irreversible":           `<span class="red">Nota:</span> Esta ação é irreversível - após uma conta ser desativada esta não pode mais ser reativada.`,
+	"disable_dex_server":             "Este servidor DEX pode ser reativado (você não terá que pagar a taxa) a qualquer momento no futuro, adicionando-o novamente.",
```
[the remark this answers is not in the document]
Comment (vctt94) [reply client/webserver/locales/pt-br.go:43]: Hey chapp, yeah it is readable, but I think: 

"Este servidor DEX pode ser reativado a qualquer momento no futuro (você não terá que pagar a taxa), adicionando-o novamente."

might be more readable

File: client/webserver/locales/pl-pl.go (hunk 1 above)
Comment (artikozel) [reply client/webserver/locales/pl-pl.go:43]: Apologies for taking so long! I'll need to make a habit out of checking my github e-mail, and soon!

Yeah, the highlighted string looks and feels okay, which is weird knowing where it came from, but if it's not broken I'm definitely not going to try and fix it.

Comment (chappjc) [reply client/webserver/locales/pl-pl.go:43]: Sounds good @artikozel.  Thanks for verifying!

File: client/webserver/locales/zh-cn.go (hunk 2 above)
Comment (LasTshaMAN) [reply client/webserver/locales/zh-cn.go]: Added `// RETRANSLATE`

**Comment by LasTshaMAN** [comment]
Followed up on all the comments and rebased

**Review by chappjc — APPROVED** [review]
Looks good.  The message reflects the new account discovery features.  Only legacy accounts should be exported to avoid fee repayment.

**Pull request merged**